README.md

```diff
@@ -41,26 +43,31 @@ Tracked Image
 
 ## Getting Started
```

Comment:
I think we should mention that all of this currently requires a GPU. So far reading this I wouldn't know that this

Reply:
it doesn't require a GPU, there is a CPU version

Reply:
Yeah definitely. If you think it's not worth mentioning the performance difference between GPU and CPU then no need to include